From 728fc07d2a4ec331eb2c6fbc7bae19767c7ade3c Mon Sep 17 00:00:00 2001
Message-ID: <728fc07d2a4ec331eb2c6fbc7bae19767c7ade3c.1699403992.git.sharlatanus@gmail.com>
In-Reply-To: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
References: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 14:45:03 +0000
Subject: [PATCH 36/67] gnu: go-github-com-emicklei-go-restful: Move to (gnu
 packages golang-web).

* gnu/packages/golang.scm (go-github-com-emicklei-go-restful): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: Idc88242d2ec3274a0b68244f5e91bbfc2dd8543c
---
 gnu/packages/golang-web.scm | 24 ++++++++++++++++++++++++
 gnu/packages/golang.scm     | 24 ------------------------
 2 files changed, 24 insertions(+), 24 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index c501f55685..4cef7b6641 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -1,5 +1,6 @@
 ;;; GNU Guix --- Functional package management for GNU
 ;;; Copyright © 2018 Pierre-Antoine Rouby <pierre-antoine.rouby@inria.fr>
+;;; Copyright © 2020 Jack Hill <jackhill@jackhill.us>
 ;;; Copyright © 2020 Martin Becze <mjbecze@riseup.net>
 ;;; Copyright © 2020 Oleg Pykhalov <go.wigust@gmail.com>
 ;;; Copyright © 2020 Ryan Prior <rprior@protonmail.com>
@@ -260,6 +261,29 @@ (define-public go-github-com-coreos-go-oidc-v3
      (list go-github-com-go-jose-go-jose-v3
            go-golang-org-x-oauth2))))
 
+(define-public go-github-com-emicklei-go-restful
+  (package
+    (name "go-github-com-emicklei-go-restful")
+    (version "3.4.0")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/emicklei/go-restful")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "0m1y5a6xr6hmdj77afrvyh2llkbhn1166lcrgis654shl8zs9qhz"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/emicklei/go-restful"))
+    (home-page "https://github.com/emicklei/go-restful")
+    (synopsis "Build REST-style web services using Go")
+    (description "This package provides @code{go-restful}, which helps
+developers to use @code{http} methods explicitly and in a way that's
+consistent with the HTTP protocol definition.")
+    (license license:expat)))
+
 (define-public go-github-com-felixge-httpsnoop
   (package
     (name "go-github-com-felixge-httpsnoop")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index bd70a4271c..30d32dcba9 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -4346,30 +4346,6 @@ (define-public go-github-com-blang-semver
       (description "Semver is a library for Semantic versioning written in Go.")
       (license license:expat))))
 
-(define-public go-github-com-emicklei-go-restful
-  (package
-    (name "go-github-com-emicklei-go-restful")
-    (version "3.4.0")
-    (source
-     (origin
-       (method git-fetch)
-       (uri (git-reference
-             (url "https://github.com/emicklei/go-restful")
-             (commit (string-append "v" version))))
-       (file-name (git-file-name name version))
-       (sha256
-        (base32
-         "0m1y5a6xr6hmdj77afrvyh2llkbhn1166lcrgis654shl8zs9qhz"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/emicklei/go-restful"))
-    (home-page "https://github.com/emicklei/go-restful")
-    (synopsis "Build REST-style web services using Go")
-    (description "This package provides @code{go-restful}, which helps
-developers to use @code{http} methods explicitly and in a way that's consistent
-with the HTTP protocol definition.")
-    (license license:expat)))
-
 (define-public go-cloud-google-com-go-compute-metadata
   (package
     (name "go-cloud-google-com-go-compute-metadata")
-- 
2.41.0

